acra/configuring-maintaining/general-configuration/acra_keymaker.md

```diff
@@ -0,0 +1,178 @@
+---
+title: AcraKeymaker
+bookCollapseSection: true
+---
+
+# AcraKeymaker
+
+## Command line flags
+
+### General flags
+
+* `--client_id=<id>`
+  
+  Use provided client ID as identifier to generate keys or keypairs.
+
+* `--tls_cert=<path>`
+  
+  Path to TLS certificate to use as client_id identifier. Works only when passed empty ClientID: `--client_id=""`.
+
+* `--tls_identifier_extractor_type={distinguished_name|serial_number}`
+  
+  Decide which field of TLS certificate to use as ClientID.
+
+  * `distinguished_name` — (default) certificate Distinguished Name (DN).
+  * `serial_number` — certificate serial number.
+
+* `--keystore=<v1|v2>`
+  
+  Set keystore format. If `--keys_output_dir` points to existing keystore destination then `acra-keymaker` automatically detects version of keystore.
+  If destination storage is empty and hasn't any key then this argument is required. Read more about [keystore versions]({{< ref "/acra/keys/versions/" >}})
```

Comment:
It would be better to make it more explicit that this needs to be used only for new keystores:

```suggestion
* `--keystore=<v1|v2>`
  
  Set keystore format.
  Read more about [keystore versions]({{< ref "/acra/keys/versions/" >}}).
  
  This option is required for newly created keystores.
  If `--keys_output_dir` points to existing keystore then `acra-keymaker` automatically detects the version of the keystore.
```